src/man/sssd.conf.5.xml

```diff
@@ -2348,6 +2348,14 @@ pam_account_locked_message = Account locked, please contact help desk.
                                  <manvolnum>5</manvolnum>
                              </citerefentry>.
                         </para>
+                        <para>
+                            <emphasis>NOTE:</emphasis> Sudo rules are
+                            periodically downloaded on the background unless
+                            sudo provider is explicitly disabled. Set
```

Comment:
I would say "unless the sudo provider", but you might want to ping @justin-stephenson to check if the English is correct.

Reply:
Agreed about "unless the sudo provider", also I would change the part before to "downloaded in the background"  - otherwise LGTM.